packages/fx-core/resource/package.nls.json

```diff
@@ -395,6 +398,12 @@
   "core.createProjectQuestion.noPlugin.detail": "Create declarative Copilot only",
   "core.createProjectQuestion.addPlugin.label": "Add plugin",
   "core.createProjectQuestion.addPlugin.detail": "Create declarative Copilot with API plugin",
+  "core.createProjectQuestion.addExistingPlugin.title": "Import from Existing - Upload files",
```

Comment:
Is this - Import from existing uploaded files? Or 'Upload files' is a CTA?

Reply:
I feel it is more for Upload files. It means for importing from an existing plugin.